databroker/v2.py

```diff
@@ -10,6 +14,32 @@ class Broker(Catalog):
 
     It includes an accessor the databroker API version 1.
     """
+
+    def __init__(self, *, handler_registry=None, root_map=None,
+                 filler_class=event_model.Filler,**kwargs):
+
+        self._root_map = root_map
+        self._filler_class = filler_class
+        if handler_registry is None:
+            handler_registry = discover_handlers()
+        self._handler_registry = parse_handler_registry(handler_registry)
+        self.handler_registry = event_model.HandlerRegistryView(
+            self._handler_registry)
+
+        self._get_filler = partial(self._filler_class,
+                              handler_registry=self.handler_registry,
+                              root_map=self._root_map,
+                              inplace=False)
```

Comment:
Consider aligning/indenting those 3 lines at the level of `self._filler_class,`.

Reply:
fixed